core/protocfg/core.go

```diff
@@ -17,6 +18,8 @@ const (
 	CfgProtocolPublicKeyRanges = "protocol.publicKeyRanges"
 	// the minimum PoW score required by the network
 	CfgProtocolMinPoWScore = "protocol.minPoWScore"
+	// the network ID on which this node operates on.
+	CfgProtocolNetworkIDName = "protocol.networkID"
```

Comment:
Why make this configurable? Why not read it out of the snapshot? Makes it more difficult to misconfigure

Reply:
As discussed in private, we're adding this config option to be analog somewhat to the coordinator address in iota 1.5.